docs/contributing/API Review Process.md

```diff
@@ -0,0 +1,48 @@
+
+4. **Owner makes decision**. When the owner believes enough information is available to make a decision, they will update the issue accordingly:
+
+    * **Mark for review**. If the owner believes the proposal is actionable, they will label the issue with `api-ready-for-review`.
+    * **Close as not actionable**. In case the issue didn't get enough traction to be distilled into a concrete proposal, the owner will close the issue.
+    * **Close as won't fix as proposed**. Sometimes, the issue that is raised is a good one but the owner thinks the concrete proposal is not the right way to tackle the problem. In most cases, the owner will try to steer the discussion in a direction that results in a design that we believe is appropriate. However, for some proposals the problem is at the heart of the design which can't easily be changed without starting a new proposal. In those cases, the owner will close the issue and explain the issue the design has.
+    * **Close as won't fix**. Similarly, if proposal is taking the product in a direction we simply don't want to go, the issue might also get closed. In that case, the problem isn't the proposed design but in the issue itself.
+
+5. **API gets reviewed**. The group conducting the review is called *RAR*, which stands for *roslyn api reviewers*. In the review, we'll take notes and provide feedback. After the review, we'll publish the notes in the [API Review repository](https://github.com/dotnet/apireviews) and at the end of the relevant issue. Multiple outcomes are possible:
+
+    * **Approved**. In this case the label `api-ready-for-review` is replaced
+    with `api-approved`.
+    * **Needs work**. In case we believe the proposal isn't ready yet, we'll
+    replace the label `api-ready-for-review` with `api-needs-work`.
+    * **Rejected**. In case we believe the proposal isn't a direction we want to go after, we simply write a comment and close the issue.
+
+## Review schedule
+
+ There are three methods to get an API review:
+
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/roslyn` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while. Progress of issues can be tracked via https://apireview.net/backlog/roslyn (PROTOTYPE: make this link actually work).
```

Comment:
Above you said the owner adds the `api-ready-for-review` label, but here it implies the issue should be logged with the label applied. This is easily fixed with a wording change, but still begs the question "How do I get an owner?"

Perhaps a label in the template like `api-needs-owner` would make sense? Or are you confident the existing triage process is enough?

EDIT: The below point suffers from the same confusion about who adds the label. Is this whole section aimed at owners?

Reply:
Yes, this section is aimed at owners (note it's mostly copied verbatim from the runtime repo).